node/sync.go

```diff
@@ -269,6 +269,35 @@ func (d *Pegnetd) SyncBlock(ctx context.Context, tx *sql.Tx, height uint32) erro
 	return nil
 }
 
+// DevelopersPayouts works along with SnapshotPayouts. We assume that the current shapshot moved to the "past", and updated the current snapshot.
+// So we can proceed to do the snapshot developer payouts.
+func (d *Pegnetd) DevelopersPayouts(tx *sql.Tx, fLog *log.Entry, height uint32, heightTimestamp time.Time) error {
```

Comment:
Where do you call `DevelopersPayouts()` from pegentd?

Reply:
DevelopersPayouts() doesn't exist in anywhere.

Reply:
@StarNeit this function runs in a snapshot mechanism from PIP12 when it's merge I'll be able to update its usage. It happens in a single run.

Reply:
@StarNeit moved logic to separate block in sync go line 272, so it won't be dependent on pip12 anymore